docs/server/multi-cluster.md

```diff
@@ -0,0 +1,394 @@
+---
+id: multi-cluster
+title: Multi-cluster Replication
+sidebar_label: Multi-cluster Replication
+---
+
+import CustomWarning from "../shared/CustomWarning.js"
+
+<CustomWarning>
+
+Temporal's Multi-cluster Replication feature is considered **experimental** and not subject to normal [versioning and support policy](/docs/server/versions-and-dependencies).
+
+</CustomWarning>
+
+This guide introduces Temporal's Multi-cluster Replication capabilities.
+You can set this up with [`clusterMetadata` configuration](/docs/server/configuration#clustermetadata), however this is only meant to be a conceptual guide rather than a detailed tutorial.
+Please reach out to us if you need to set this up.
+
+## Overview
+
+Multi-cluster Replication is a feature which asynchronously replicates workflows from active cluster to other passive clusters, for backup & state reconstruction.
+When necessary, the customer can failover to any of the clusters which have the backup for high availability.
+
+In terms of the CAP theorem, Multi-cluster Replication adds high availability to Temporal's strongly consistent system.
```

Comment:
this seems to claim C & A & P, which is not the case.

Reply:
can you suggest better wording? original wording was "Cadence Cross DC is an AP (in terms of CAP)." - but [here](https://docs.temporal.io/docs/concepts/introduction/#modern-solution) we claim that it is CP

Reply:
single DC setup is CP
cros DC setup is AP

Reply:
i would just delete this line

Reply:
```suggestion
```